Assets/Scripts/PlayerScript.cs

```diff
@@ -280,15 +284,15 @@ public Weapon GetActiveWeapon()
                     smallestDistance = distance;
                     closestItem = item;
                 }
+                if (item.selected == true){
+                    item.Deselect();
+                }
             }
-            if (smallestDistance < closestItem.selectionRadius & closestItem.selected == false){
+            this.selectedInteractable = null;
+            if (smallestDistance < closestItem.selectionRadius){
```

Comment:
This might seem picky but can we move this `selectionRadius` property onto the player and rename it to `interactionRadius`. My reasoning is that I don't think we will ever want different interaction radii for different objects, and currently this method wouldn't support that anyway (as it only finds the closest item it won't account for an item further away with a larger interaction radius)